KopernicusExamples/Creating New Bodies/OceanBody/Oceanus.cfg

```diff
@@ -26,7 +32,7 @@
         }
         Orbit
         {
-            referenceBody = Bastion
+            referenceBody = Sun
```

Comment:
When you remove Bastion the body starts orbiting the sun now, but with the same semiMajorAxis as if it would orbit around Bastion. Which means that it most likely spawns inside of the sun.

To be honest I don't think the dependency on Bastion is that much of a deal.